packaging/irods_rest.py

```diff
@@ -58,6 +58,38 @@ def authenticate(_user_name, _password, _auth_type):
 
     return body
 
+def lp_rename(_token, _src, _dest):
```

Comment:
`lp_rename` -> `rename_logical_path`

Using more letters provides more context for the reader.